packages/amplify-graphql-schema-generator/src/ts-schema-generator/generate-ts-schema.ts

```diff
@@ -18,3 +19,7 @@ export const generateTypescriptDataSchema = (schema: Schema, config?: DataSource
   );
   return result;
 };
+
+
+
+
```

Comment:
Run `yarn prettier --write .` from the top-level project dir to fix this and other linting errors before submitting.